src/RWS.jl

```diff
@@ -230,5 +224,21 @@ function BuildRWS(G::Groups.FPGroup;
         rws.orderingset = true # This was not in the original file
     end
 
+    #=
+        mutable struct FPGroup <: AbstractFPGroup
+            gens::Vector{FPSymbol}
+            rels::Dict{FPGroupElem, FPGroupElem}
+        end
+    =#
+
+    # Generators
+    C = [""]
+    append!(C, string.(G.gens))
+    GC.@preserve C true
+    # Get the number of generators
+    rws.num_gens = size(C)[1] - 1
+    # Move the generators to the RWS. Warning. These are not processed and are assumed to be correct.
+    rws.gen_name = Base.unsafe_convert(Ptr{Ptr{Int8}}, Base.cconvert(Ptr{Ptr{Int8}}, C))
```

Comment:
This is the problematic part of the code. I need to prepend one element to the `G.gens` array, so it is correctly allocated on the `C`-side. However, the object will be removed after the call ends (usually in less than a few seconds after the execution, as I experimentally found out).